### PR TITLE
fix: typo in log message, keep code style consistency

### PR DESCRIPTION
**Pull request by luckymrwang**
_Provide a description of what has been changed_

### Checklist

- [x] When introducing a new scaler, I agree with the [scaling governance policy](https://github.com/kedacore/governance/blob/main/SCALERS.md)
- [x] I have verified that my change is according to the [deprecations & breaking changes policy](https://github.com/kedacore/governance/blob/main/DEPRECATIONS.md)
- [x] Tests have been added
- [x] Changelog has been updated and is aligned with our [changelog requirements](https://github.com/kedacore/keda/blob/main/CONTRIBUTING.md#Changelog)
- [x] A PR is opened to update our Helm chart ([repo](https://github.com/kedacore/charts)) *(if applicable, ie. when deployment manifests are modified)*
- [x] A PR is opened to update the documentation on ([repo](https://github.com/kedacore/keda-docs)) *(if applicable)*
- [x] Commits are signed with Developer Certificate of Origin (DCO - [learn more](https://github.com/kedacore/keda/blob/main/CONTRIBUTING.md#developer-certificate-of-origin-signing-your-work))

Fixes #

Relates to #

### CODE REVIEW

**Review by JorTurFer — APPROVED** [review]
Thanks for the stuff style fix!

File: controllers/keda/hpa.go [hunk 1]
```diff
@@ -63,7 +63,7 @@ func (r *ScaledObjectReconciler) createAndDeployNewHPA(ctx context.Context, logg
 
 	err = kedastatus.UpdateScaledObjectStatus(ctx, r.Client, logger, scaledObject, status)
 	if err != nil {
-		logger.Error(err, "Error updating scaledObject status with used hpaName")
+		logger.Error(err, "Failed to update scaledObject status with used hpaName")
```
Comment (zroubalik) [inline controllers/keda/hpa.go:66]: I know it is already merged, but go error messages should start with lowercase, we should slowly reflect that in the codebase when we are doing a new changes.

Comment (luckymrwang) [reply controllers/keda/hpa.go:66]: ok

**Pull request merged**